developer-docs-site/docs/tutorials/your-first-nft-sdk.md

```diff
@@ -0,0 +1,372 @@
+---
+title: "Your First NFT using the SDK"
```

Comment:
Why do we adding a new first_nft with SDK ?  this is what it looks like on website now. 
![Screen Shot 2022-08-24 at 5 46 42 PM](https://user-images.githubusercontent.com/1082769/186548920-2ec3db9b-074e-4186-88b3-ad1d41760c50.png)

Reply:
So we are now in the process of enhancing all tutorials to use SDK, as opposed to REST API-heavy tutorials we currently have. During the transition we'll see both. I will gracefully remove the non-SDK ones after some time.

Reply:
sounds great. But the two are not the same. A lot of content in the original first_nft is not in the SDK ones.